src/test/resources/features/per_project_configuration.feature

```diff
@@ -26,3 +26,26 @@ Feature: Per-project configuration
 		Then I see "Using groovy version 2.4.1 in this shell."
 		And the candidate "groovy" version "2.4.1" should be in use
 		And the candidate "groovy" version "2.0.5" should be the default
+
+	Scenario: The env clear subcommand is issued without an active project configuration
+		Given the system is bootstrapped
+		When I enter "sdk env clear"
+		Then I see "No environment currently set!"
+		And the exit code is 1
+
+	Scenario: The env clear subcommand is issued and the active project configuration is missing
+		Given the system is bootstrapped
+		And a project configuration is active and points to "/tmp"
```

Comment:
We should be using a random temporary directory here like in other tests to prevent a clash in the residual state.

Reply:
I didn't know about that random temp directory. Much better like that. Done in 108b03f